### PR TITLE
chore: Make it well-defined how to run grapher without wordpress

### PR DESCRIPTION
**Pull request by larsyencken**
Currently, grapher fails to render some pages (e.g. `/grapher/:slug`) if you do not have a wordpress database dump set up and if you do not have a working wordpress API url with credentials.

This PR makes it well defined how to disable these two Wordpress integrations by blanking out specific config keys (`WORDPRESS_DB_NAME` and `WORDPRESS_API_URL`). When they are disabled, the site runs without them and does not attempt operations that would have failed. This makes rendering local charts much much faster.

For example, the http://localhost:3030/admin/test/embeds page rapidly renders all local charts after this change, whereas before they blocked on API calls to a Wordpress instance and were slower than the production site.

### CODE REVIEW

**Comment by larsyencken** [comment]
Sorry, added another change to make a few more routes succeed

**Comment by danielgavrilov** [comment]
It's great to decouple these Wordpress dependencies from Grapher!

As mentioned on our call, there is one point where Wordpress gets called from Grapher Admin – retrieving the posts that reference this chart in the "Refs" tab when editing a chart: https://github.com/owid/owid-grapher/blob/9bc2033bbdc0e834d0d8589a080232d602e6f84e/adminSiteServer/apiRouter.ts#L346-L351

It would be good to have an if statement there as well for consistency, instead of the try/catch block.

Also, would be good if @mlbrgl does a quick review on this, he might know some endpoints/implications that I'm not aware of. 

Also, maybe `WORDPRESS_DB_NAME` and `WORDPRESS_API_URL` can be blank by default? Although this might need everyone to change their `.env` so might be better to do separately and not block this PR.

**Comment by larsyencken** [comment]
@danielgavrilov Patched the reference check to return an empty list when Wordpress database is disabled, and also patched the default settings so that Wordpress is not assumed.

**Comment by MarcelGerber** [comment]
Makes sense to me, too!
But I also agree that ideally @mlbrgl should have a look at this, too.

**Review by mlbrgl — APPROVED** [review]
Nice!

You might want to apply the same treatment to getBlockContent: https://github.com/owid/owid-grapher/blob/d4e9ec754196c438982fdd316265e79ad25487e5/db/wpdb.ts#L577

This gets called on Explorer pages.

Otherwise, looks good to me!

File: db/wpdb.ts
Comment (larsyencken) [reply db/wpdb.ts]: Done!

**Pull request merged**
